puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -130,6 +130,14 @@
   }
   File['/etc/neutron/plugin.ini'] -> Exec['neutron-db-manage upgrade']
 
+  # short-term workaround for BZ 1181592
+  package{'bind-utils': } ->
+  exec {'neutron-etc-hosts-workaround':
+    command => 'dig A $(hostname) | grep -A1 "ANSWER SEC" | tail -n 1 | awk \'{print $NF " " $1}\' | sed -e \'s/.$//g\' >>/etc/hosts',
```

Comment:
Can we somehow check that the entry was included?

A resolution failure would fail silently to include the entry, although I suppose later runs would retry due to the "unless"